storybook/stories/API/cartesian/Brush.stories.tsx

```diff
@@ -58,6 +58,10 @@ const GeneralProps: Args = {
     description: 'The handler of changing the active scope of brush.',
     table: { type: { summary: 'Function' }, category: 'General' },
   },
+  onDragEnd: {
+    description: 'The handler of ending the brush drag.',
+    table: { type: { summary: 'Function' }, category: 'General' },
```

Comment:
Should both possibly be in the category `Event Handler`? What do you think?

Reply:
Yes, i was thinking about that. I wanted to make a new PR for fixing the groups. Will update this now, and can make another contribution later.